src/components/speakers/SpeakerDirectory.tsx

```
@@ -0,0 +1,205 @@
+                                ) : (
+                                    <ul className="space-y-2">
+                                        {selectedSpeaker.events.map((event) => (
+                                            <li key={event.event_id} className="text-xs">
+                                                <Link
+                                                    to={`/events/${event.event_id}`}
+                                                    className="font-medium text-indigo-600 hover:underline dark:text-indigo-400"
+                                                >
+                                                    {event.event_title}
+                                                </Link>
+                                                <p className="text-slate-500 dark:text-slate-400">
+                                                    {new Date(event.event_date).toLocaleDateString()} • {event.club_name}
```

ESLint

[error] Delete `··························`

(prettier/prettier)